src/enc/impls.rs

```diff
@@ -283,12 +283,12 @@ impl Encode for char {
     }
 }
 
-impl Encode for &'_ [u8] {
-    fn encode<E: Encoder>(&self, encoder: &mut E) -> Result<(), EncodeError> {
-        super::encode_slice_len(encoder, self.len())?;
-        encoder.writer().write(self)
-    }
-}
+// impl Encode for &'_ [u8] {
```

Comment:
More commented out code to burn with holy fire :fire:

Reply:
Actually, put this back please. Comment out the generic T

Reply:
I added a BlockedTODO for specialization, which is why I wanted to keep this code